src/main/views/includes/macros/partyDetails.njk

```diff
@@ -74,7 +74,7 @@
     }}
   {% endif %}
 
-  {% if (partyDetails.type === 'individual' and partyDetails.dateOfBirth) %}
+  {% if (partyDetails.type === 'individual' and partyDetails.dateOfBirth and partyDetails.dateOfBirth.known ) %}
```

Comment:
Need to ask @BenClancy what to do if DOB is not known. It hides it for now however we could say `Not sure` for example.